pygments/lexers/qlik.py

```diff
@@ -0,0 +1,139 @@
+from pygments.token import *
+from pygments.lexers._qlik_builtins import (
+    OPERATORS_LIST,
+    STATEMENT_LIST,
+    SCRIPT_FUNCTIONS,
+    CONSTANT_LIST,
+)
+
+__all__ = ["QlikLexer"]
+
+field_name_options = r"(\w|\[[\w\s]\]|\"[\w\s]\")"
+
+
+class QlikLexer(RegexLexer):
+    """
+    Lexer for qlik code, including .qvs files
+    """
+
+    name = "Qlik"
+    aliases = ["qlik", "qlikview", "qliksense", "qlikscript"]
+    filenames = ["*.qvs", "*.qvw"]
+
+    flags = re.IGNORECASE
+
+    tokens = {
+        # Handle multi-line comments
+        "comment": [
+            (r"\*/", Comment.Multiline, "#pop"),
+            (r"[^\*/]+", Comment.Multiline),
+        ],
+        # Handle numbers
+        "numerics": [
+            (r"\b[0-9][0-9]*\.[0-9]+([eE][0-9]+)?[fd]?\b", Number.Float),
+            (r"\b[0-9]+\b", Number.Integer),
+        ],
+        # Handle variable names in things
+        "interp": [
+            (r"(\$\()(\w+)(\))", bygroups(Keyword, Name.Variable, Keyword)),
+        ],
+        # Handle strings
+        "string": [
+            (
+                r"(\')",
+                String,
+                "#pop",
+            ),
+            include("interp"),
+            (r"[^\']+", String),
+        ],
+        #
+        "assignment": [
+            (r";", Punctuation, "#pop"),
+            include("root"),
+        ],
+        "field_name_quote": [
+            (
+                r"(\")",
+                Keyword,
+                "#pop",
+            ),
+            include("interp"),
+            (r"[^\"]+", Keyword),
+        ],
+        "field_name_bracket": [
+            (
+                r"(\])",
+                Keyword,
+                "#pop",
+            ),
+            include("interp"),
+            (r"[^\]]+", Keyword),
+        ],
+        "function": [(r"\)", Operator.Word, "#pop"), include("root")],
+        "root": [
+            # Whitespace and comments
+            (r"\s+", Text.Whitespace),
+            (r"/\*", Comment.Multiline, "comment"),
+            (r"//.*\n", Comment.Singleline),
+            # variable assignment
+            (
+                r"(let|set)(\s+)",
+                bygroups(
+                    Keyword.Declaration,
+                    Text.Whitespace,
+                ),
+                "assignment",
+            ),
+            # Word operators
+            (
+                words(OPERATORS_LIST["words"], prefix=r"\b", suffix=r"\b"),
+                Operator.Word,
+            ),
+            # Statements
+            (
+                words(
+                    STATEMENT_LIST,
+                    suffix=r"\b",
+                ),
+                Keyword,
+            ),
+            # Table names
+            (r"[a-z]\w*:", Keyword.Declaration),
+            # Constants
+            (words(CONSTANT_LIST, suffix=r"\b"), Keyword.Constant),
+            # Functions
+            (words(SCRIPT_FUNCTIONS, suffix=r"\s*\("), Operator.Word, "function"),
```

Comment:
The suffix is going to match whitespace and parentheses as part of the function name. You need a lookahead assertion: `r"(?=\s*\("`.

Also, it should probably be `Name.Builtin` or similar, not `Operator.Word`.